src/main/java/teammates/storage/entity/FeedbackSession.java

```diff
@@ -381,4 +381,12 @@ public String toString() {
                 + ", isPublishedEmailEnabled=" + isPublishedEmailEnabled + "]";
     }
 
+    private String convertOffsetToZoneId(double offset) {
+        return ZoneId.ofOffset("UTC", ZoneOffset.ofTotalSeconds((int) (offset * 60 * 60))).getId();
+    }
+
+    private double convertZoneIdToOffset(String zoneId) {
+        return ((double) ZoneId.of(zoneId).getRules().getOffset(Instant.now()).getTotalSeconds()) / 60 / 60;
```

Comment:
Unrelated to this PR, but now that you're using this `ZoneId`, you need to check and sync with the ones defined in `moment-timezone` (we have `TimezoneSyncerTest` exactly for that purpose). I ran an experiment some time back and found some discrepancies in `moment-timezone` version `0.5.13`, and even worse for `0.5.14` (current latest).